pkgs/by-name/os/osu-lazer/package.nix

```
@@ -68,8 +69,9 @@ buildDotnetModule rec {
   fixupPhase = ''
     runHook preFixup
 
-    wrapProgram $out/bin/osu! \
-      --set OSU_EXTERNAL_UPDATE_PROVIDER 1
+    wrapProgram $out/bin/osu! ${
+      lib.optionalString nativeWayland "--set SDL_VIDEODRIVER=wayland"
```

> Builder called die: makeWrapper doesn't understand the arg OSU_EXTERNAL_UPDATE_PROVIDER

The `=` sign messed you up
```suggestion
      lib.optionalString nativeWayland "--set SDL_VIDEODRIVER wayland"
```